src/summary.rs

```diff
@@ -385,7 +394,8 @@ mod tests {
         )
         .await
         .unwrap();
-        assert_summary_texts(&msg, ctx, "👤 Contact").await;
+        chat_id.set_draft(ctx, Some(&mut msg)).await.unwrap();
+        assert_summary_texts(&msg, ctx, "👤 Alice Wonderland").await;
```

Comment:
Would be nice to also test somewhere that the summary is shown correctly after sending it (`ctx.send_msg(chat_id, &mut msg)`)

Reply:
Added a check to `test_receive_vcard()` as it already sends vCards